Extension/src/LanguageServer/ui.ts

```diff
@@ -426,12 +435,16 @@ export class LanguageStatusUI {
     private showConfigureIntelliSenseButton: boolean = false;
     private configureIntelliSenseTimeout?: NodeJS.Timeout;
 
-    public async ShowConfigureIntelliSenseButton(show: boolean, client?: Client): Promise<void> {
+    public async ShowConfigureIntelliSenseButton(show: boolean, logTelemetry: boolean, client?: Client, configurationType?: ConfigurationType, sender?: string): Promise<void> {
+        if (configurationType !== undefined && sender !== undefined && logTelemetry) {
```

Comment:
I don't think you need a `logTelemetry` boolean for this function.  Is there a case where you would pass in the configurationType and the sender and not want to log it?

Reply:
There was one but we went back on it but it could be useful to keep the boolean just because there may be cases in the future where we want to eliminate redundant telemetry

Reply:
Please remove it.  It serves no purpose today and is easy to add later if we ever do actually need it.